tests/integration/check-custom-repo/test_custom-repos.py

```diff
@@ -0,0 +1,67 @@
+import re
+import sys
+
+from collections import namedtuple
+
+import pexpect
+import pytest
+
+
+try:
+    from pathlib import Path
+except ImportError:
+    from pathlib2 import Path
+
+from envparse import env
+
+
+def get_system_version(system_release_content=None):
+    """Return a namedtuple with major and minor elements, both of an int type.
+
+    Examples:
+    Oracle Linux Server release 6.10
+    Oracle Linux Server release 7.8
+    CentOS release 6.10 (Final)
+    CentOS Linux release 7.6.1810 (Core)
+    CentOS Linux release 8.1.1911 (Core)
+    """
+    match = re.search(r".+?(\d+)\.(\d+)\D?", system_release_content)
+    if not match:
+        return "not match"
+    version = namedtuple("Version", ["major", "minor"])(int(match.group(1)), int(match.group(2)))
+
+    return version
+
+
+@pytest.mark.good_tests
+def test_good_convertion_without_rhsm(shell, convert2rhel):
+    """
+    Test if --enablerepo are not skipped when  subscription-manager are disabled and test if repo passed are valid.
+    """
+    with open("/etc/system-release", "r") as file:
+        system_release = file.read()
+        system_version = get_system_version(system_release_content=system_release)
+        if system_version.major == 7:
+            repo = "rhel-7-server-rpms"
+        elif system_version.major == 8:
+            repo = "rhel-8-for-x86_64-baseos-rpms"
+
+    with convert2rhel("-y --no-rpm-va --disable-submgr --enablerepo {} --debug".format(repo)) as c2r:
```

Comment:
```suggestion
            enable_repo_opt = "--enablerepo rhel-7-server-rpms"
        elif system_version.major == 8:
            enable_repo_opt = "--enablerepo rhel-8-for-x86_64-baseos-rpms --enablerepo rhel-8-for-x86_64-appstream-rpms"

    with convert2rhel("-y --no-rpm-va --disable-submgr {} --debug".format(enable_repo_opt)) as c2r:
```
I'm surprised that the conversion passed with only BaseOS enabled. It used to be necessary to enable both BaseOS and AppStream.

Reply:
But if you're going to make the test faster by sending Ctrl+C, it does not matter and you can leave this as is.